src/app/legacy/components/MediaPlayer/Placeholder/index.jsx

```diff
@@ -3,9 +3,24 @@ import styled from '@emotion/styled';
 import { string, func, shape, oneOf } from 'prop-types';
 import Image from '#psammead/psammead-image/src';
 import PlayButton from '#psammead/psammead-play-button/src';
-import { C_POSTBOX } from '#psammead/psammead-styles/src/colours';
+import {
+  C_POSTBOX,
+  C_BLACK,
+  C_WHITE,
```

Comment:
Just to add on to this, you can access the theme within the styled API instead of importing by destructuring the theme object, something like this:
```css
&:focus-visible::before {
    ${placeholderOutline}
    ${({ theme: { palette } }) =>
      `box-shadow: 0 0 0 ${focusIndicatorThickness} ${palette.WHITE} inset;
       border: ${focusIndicatorThickness} solid ${palette.BLACK};`}
```

Reply:
Great thank you! I have updated the files to use `import { BLACK, WHITE } from '#app/components/ThemeProvider/palette'`

When I tried your suggestion @HarveyPeachey the unit tests failed with an error message 'TypeError: Cannot read properties of undefined...' I assume I'd need to update something in the tests, or I'm missing an import somewhere. If you have an example of a component I could look at, or any advice, at that would be really helpful!

Reply:
Ahh yeah it's because the tests don't have access to the context, yeah just leave it it would require a lil bit more work to get it working for legacy tests @Isabella-Mitchell

Reply:
Cool thanks @HarveyPeachey! I know there is a ticket in the backlog for updating all the colours. So it's good to know this method - though I think updating the tests should fall into the scope of that ticket.